File: Pythonleetcode/Leetcode268.py
class Solution(object):
    def missingNumber(self, nums):


        nums.sort()
        for i in range(0,len(nums)-1):
            if nums[i+1]-nums[i]==2:
                return nums[i]+1
        if max(nums)==len(nums):
            return 0
        else: return len(nums)



print(Solution().missingNumber([3,0,1]))

File: CppPractice/Project1/Project1/Leetcode268.cpp
#include <string>

#include <vector>

#include <iostream>

#include <stdlib.h>

#include <time.h>

#include <sstream>

#include <algorithm>

#include <map>

#include <bitset>

using namespace std;


class Solution {
public:
    int missingNumber(vector<int>& nums) {
        sort(nums.begin(),nums.end());
        for (int i = 0; i < nums.size(); i++)
        {
            if (nums[i]!=i)
            {
                return  i;
            }

        }
        return nums.size();

    }
};